docs/using-scylla/cassandra-compatibility.rst

```diff
@@ -2,51 +2,63 @@
+Apache Cassandra.  See :doc:`Scylla Features </using-scylla/features>` for more information about ScyllaDB features.
 
-   This table includes Apache Cassandra features and the corresponding Scylla Open Source status. It does **not** include Scylla Enterprise features or Scylla Open Source features with no match in Apache Cassandra.
-   More on :doc:`Scylla Features </using-scylla/features>`
+How to Read the Tables on This Page
+-------------------------------------
 
-For all tables below, the following convention is used:
-
-* |x| - Not Available in Scylla
-* NC - Available in Scylla, but not compatible with Apache Cassandra
-* |v| - Available and compatible with Apache Cassandra
+* |v| - Available in ScyllaDB and compatible with Apache Cassandra.
+* |x| - Not available in ScyllaDB.
+* **NC** - Available in ScyllaDB, but not compatible with Apache Cassandra.
 
 Interfaces
 ----------
 
-==============================  =====================================================  =============
-Apache Cassandra                Version                                                Comments
-==============================  =====================================================  =============
-CQL                             CQL spec |cql-version|, Native protocol v4 (C* 3.11)   More below
-------------------------------  -----------------------------------------------------  -------------
-Thrift                          Compatible with Cassandra 2.1
-------------------------------  -----------------------------------------------------  -------------
-SSTable format (all versions)   3.11(mc / md ), 2.2(la), 2.1.8 (ka)                    md format is supported in Scylla 4.3, 2021.1.0 and later
-------------------------------  -----------------------------------------------------  -------------
-JMX                             3.11                                                   More Below
-------------------------------  -----------------------------------------------------  -------------
-Configuration (cassandra.yaml)	3.11
-------------------------------  -----------------------------------------------------  -------------
-Log                             NC
-------------------------------  -----------------------------------------------------  -------------
-Gossip and internal streaming	NC
-------------------------------  -----------------------------------------------------  -------------
-SSL                             NC
-==============================  =====================================================  =============
+.. list-table::
+   :widths: 25 45 30
+   :header-rows: 1
+
+   * - Apache Cassandra Interface
+     - Version Supported by ScyllaDB
+     - Comments
+   * - CQL
+     - | Fully compatible with version 3.3.1, with additional features from later CQL versions (for example, :ref:`Duration type <durations>`).
+       | Fully compatible with protocol v4, with additional features from v5.
```

Comment:
I'm not an expert in this, but how do these "additional features" work? A client that believes it is using protocol v4 can't use any features from v5...

Reply:
@psarna Could you answer the question asked by @nyh and suggest how this section can be improved?

Reply:
@psarna Could you help?

Reply:
They can, and do. The only example I know of is the Duration type (https://github.com/apache/cassandra/blob/387c274a624c2a85036e909af1d734d87ee00541/doc/native_protocol_v5.spec#L860), not present in v4. Users are free to use this type in CQL queries, and drivers tend to be able to recognize it too, because they often support v5, or at least parts of it. For instance, our Rust driver knows about the Duration type, even though it generally targets v4 protocol: https://github.com/scylladb/scylla-rust-driver/blob/4f5a43dd86ba3f99d722f01384d736af3f52e3b9/scylla-cql/src/frame/response/result.rs#L53

Reply:
@psarna Thanks! I think we can leave it as is.